ADOPTERS.md

```diff
@@ -0,0 +1,3 @@
+# Adopters
+
+This is a list of adopters of using PowerShell Script Analyzer in production or in their products (in alphabetical order):
```

Comment:
```suggestion
This is a list of adopters of using PowerShell Script Analyzer in production or in their products (in alphabetical order):

- [PowerShellEditorServices](https://github.com/PowerShell/PowerShellEditorServices), the official Language Server Protocol implementation for PowerShell, powering the PowerShell extensions for VSCode, [neo]vim and IntelliJ.
```

Reply:
Should make VSCode, Neovim, and IntelliJ hyperlinks.  Also, was going to standardize on asterisk instead of dash for the bullet.  In any case, I think this can be a subsequent PR once we get this one in.  I'm going to add an example as a comment.